packages/terra-clinical-result/docs/README.md

```diff
@@ -0,0 +1,119 @@
+# Terra Clinical Result
+
+The Terra Clinical Result package is a collection of standardized views for presenting clinical results documented to a Patient's Medical Chart,
+such as Vital Signs, Laboratory Results, and Discretely Documented Values. It also includes additional supporting views where clinical results
+are commonly displayed like flowsheets, result detail viewers, etc.
+
+This default Clinical Result component is intended for displaying a single result value.
+
+## Getting Started
+
+- Install with [npmjs](https://www.npmjs.com):
+```
+$ npm install terra-clinical-result
+```
+
+<!-- AUTO-GENERATED-CONTENT:START Peer Dependencies -->
+<!-- AUTO-GENERATED-CONTENT:END -->
+
+## Component Features
+
+* [Cross-Browser Support](https://github.com/cerner/terra-ui/blob/master/src/terra-dev-site/contributing/ComponentStandards.e.contributing.md#cross-browser-support)
+* [Responsive Support](https://github.com/cerner/terra-ui/blob/master/src/terra-dev-site/contributing/ComponentStandards.e.contributing.md#responsive-support)
+* [Mobile Support](https://github.com/cerner/terra-ui/blob/master/src/terra-dev-site/contributing/ComponentStandards.e.contributing.md#mobile-support)
+* [LTR/RTL Support](https://github.com/cerner/terra-ui/blob/master/src/terra-dev-site/contributing/ComponentStandards.e.contributing.md#ltr--rtl-support)
+
+## Usage Guide
+The clinical result is constructed by providing structured object with a specific construction that follows a similar pattern to the [HL7 FHIR Observation](https://www.hl7.org/fhir/observation.html) standard.
+```jsx
+/* ------ Structure for the resultData object for the standard single Clinical Result ------ */
+
+resultData = {
```

Comment:
This data structure is out of date and should be updated.

Reply:
https://github.com/cerner/terra-clinical/pull/574/commits/c7bf5367680411f6e9e0ded4ef883de6fd3b2b33